app/src/main/scala/org/alephium/explorer/persistence/schema/OutputSchema.scala

```diff
@@ -40,6 +41,8 @@ object OutputSchema extends SchemaMainChain[OutputEntity]("outputs") {
     def lockTime: Rep[Option[TimeStamp]] = column[Option[TimeStamp]]("lock_time")
     def outputOrder: Rep[Int]            = column[Int]("output_order")
     def txOrder: Rep[Int]                = column[Int]("tx_order")
+    def spentFinalized: Rep[Option[Transaction.Hash]] =
+      column[Option[Transaction.Hash]]("spent_finalized", O.Default(None))
```

Comment:
Do we use the value `Transaction.Hash` somewhere? Or do we plan to use it later?

Reply:
we currently recompute it when we want to find where the output is spent, when requesting a tx, I want latter to be able to reuse this instead of joining on inputs.